prowler/lib/outputs/compliance/mitre_attack_aws.py

```diff
@@ -36,10 +36,10 @@ def write_compliance_row_mitre_attack_aws(
             attributes_categories += attribute.Category + ", "
             attributes_values += attribute.Value + ", "
             attributes_comments += attribute.Comment + ", "
-        attributes_aws_services = attributes_aws_services[:-2]
-        attributes_categories = attributes_categories[:-2]
-        attributes_values = attributes_values[:-2]
-        attributes_comments = attributes_comments[:-2]
+        attributes_aws_services = attributes_aws_services.rstrip(", ")
+        attributes_categories = attributes_categories.rstrip(", ")
+        attributes_values = attributes_values.rstrip(", ")
+        attributes_comments = attributes_comments.rstrip(", ")
```

Comment:
What if you do a `", ".join(<attribute>`, maybe it's easier within the loop and with that you won't need the `rstrip`. Up to you, just an improvement.

Please, open up another PR to fix this in V3, thanks!